package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "postinstall": "dest=/usr/local/share/zsh/site-functions; mkdir -p $dest && ln -sf \"$PWD/pure.zsh\" $dest/prompt_pure_setup && ln -sf \"$PWD/async.zsh\" $dest/async || echo 'Could not automagically symlink the prompt. Check out the readme on how to do it manually: https://github.com/sindresorhus/pure#manually'"
+    "postinstall": "dest=\"/usr/local/share/zsh/site-functions\"; mkdir -p $dest && ln -sf \"$PWD/pure.zsh\" \"$dest/prompt_pure_setup\" && ln -sf \"$PWD/async.zsh\" \"$dest/async\" || echo 'Could not automagically symlink the prompt. Check out the readme on how to do it manually: https://github.com/sindresorhus/pure#manually'"
```

Comment:
When I tried to install at first, I got double slash: `/usr/local/share/zsh/site-functions//prompt_pure_setup` and it'd fail. No idea why though..